docs/typeformmixin.rst

```diff
@@ -0,0 +1,81 @@
+=============
+TypeformMixin
+=============
+
+To process a typeform in Django just the way you would process a Django form,
+use the ``TypeformMixin``. It requires the typeform to be mirrored by a corresponding
+Django form, question by question, but can also contain processing logic.
+To turn Typeform features on for a Django form just use the Mixin and specify
+the ``typeform_url``:
+
+.. code-block:: python
+
+    from django_typeforms.forms import TypeformMixin
+
+    from forms import MyForm
+
+    class MyTypeForm(TypeformMixin, MyForm)
+        typeform_url = 'https://xxxx.typeform.com/to/xxxxxx'
+
+``MyTypeForm`` will be a fully fledged Django form and at the same
+time be linked to the Typeform with given url.
+
+For this link to work, the ``Question reference`` for each Typeform
+question must be set to the matching Django field name, e.g. ``first_name`` for a
+field ``first_name = forms.CharField(max_length=40, blank=True)``. **This requires editiing
+all questions of a typeform and as Typeform warns might break
+other integrations.**
+
+Additionally, the typeform needs a **hidden field** "typeformuid" which
```

Comment:
This looks looks like it's meant to be `typeform_uid`

Reply:
It turns out that typeform does not allow underscores in field names. I removed them everywhere for typeformuid